doc/docgen.zig

```diff
@@ -666,6 +667,8 @@ fn genToc(allocator: *Allocator, tokenizer: *Tokenizer) !Toc {
                         source_type = SyntaxBlock.SourceType.zig;
                     } else if (mem.eql(u8, source_type_str, "c")) {
                         source_type = SyntaxBlock.SourceType.c;
+                    } else if (mem.eql(u8, source_type_str, "peg")) {
+                        source_type = SyntaxBlock.SourceType.c;
```

Comment:
If you do choose to use `syntax_block` for the grammar, this should be `SyntaxBlock.SourceType.peg`.